libchewing 0.8.3 cmake build failure

👋 trying to build the latest release, but run into some build issue. The error log is as below:


<details>
<summary>error build log</summary>

```
  CMake Error at build/data/cmake_install.cmake:48 (file):
    file INSTALL cannot find
    "/tmp/libchewing-20240526-8184-ure9ze/libchewing-0.8.3/build/data/pinyin.tab":
    No such file or directory.
  Call Stack (most recent call first):
    build/cmake_install.cmake:47 (include)
```


</details>

full build log, https://github.com/Homebrew/homebrew-core/actions/runs/9245173161/job/25431336495?pr=172810
relates to Homebrew/homebrew-core#172810

--- a/NEWS
+++ b/NEWS
@@ -2,8 +2,9 @@ What's New in libchewing 0.8.4 (May 27, 2024)
 ---------------------------------------------------------
 
 * Bug fixed
-  - Config options were incorrectly reset after certain operations (introduced
+  - Config options were incorrectly reset after certain operations. (introduced
   in v0.8.0)
+  - Installation failure if build with testing off. (introduced in v0.8.3)
 
 
 What's New in libchewing 0.8.3 (May 25, 2024)

--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -14,9 +14,9 @@ add_custom_target(data ALL DEPENDS ${ALL_DATA})
 
 # We need to copy static data to binary tree when using out of tree build.
 set(ALL_STATIC_DATA
-    ${DATA_BIN_DIR}/pinyin.tab
-    ${DATA_BIN_DIR}/swkb.dat
-    ${DATA_BIN_DIR}/symbols.dat
+    ${DATA_SRC_DIR}/pinyin.tab
+    ${DATA_SRC_DIR}/swkb.dat
+    ${DATA_SRC_DIR}/symbols.dat
 )
 
 add_custom_target(all_static_data

